package.json

```diff
@@ -127,6 +116,7 @@
     "nw-builder": "^3.1.2",
     "run-sequence": "^1.1.5",
     "stylus": "^0.54.2",
+    "webpack": "^4.5.0",
```

Comment:
that's not needed by itself

Reply:
Did you tried to boot up this project from scratch?